# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webcodecs/DecoderTypes.h
# Commit: 279a5522919a
# Full Hash: 279a5522919ac625c1f35151dc8d70fccfb4b8a1
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-12-23 09:52:03
# Regressor Bug: 1749047
# File Overlap Count: 1
# Description:
#   Bug 1749047 - Add the ability to print a VideoInfo and a VideoDecoderConfigInternal. r=chunmin,media-playback-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D196336
# ==============================================================================

diff -r d43f0c82f214 -r 279a5522919a dom/media/webcodecs/DecoderTypes.h
--- a/dom/media/webcodecs/DecoderTypes.h	Fri Dec 22 21:39:16 2023 +0000
+++ b/dom/media/webcodecs/DecoderTypes.h	Fri Dec 22 21:39:16 2023 +0000
@@ -45,7 +45,7 @@
       const VideoDecoderConfig& aConfig);
   ~VideoDecoderConfigInternal() = default;
 
-  nsString ToString();
+  nsString ToString() const;
 
   bool Equals(const VideoDecoderConfigInternal& aOther) const {
     if (mDescription.isSome() != aOther.mDescription.isSome()) {